### ISSUE TITLE
tree.match is not a function

### ISSUE THREAD

**Issue by ineptian** [maizzle/framework#519]
- Maizzle Version: 3.6.1
- Node.js Version: 15.5.0
- Windows 10

When installing a new maizzle project, and running `maizzle serve` I get met with the following error:
`Failed to compile template: transactional.html`
`tree.match is not a function`

If I delete transactional.html, it will throw the error on promotional.html instead and vice versa.

Any ideas what could be causing this? Maizzle has worked for me previously so not sure exactly what the deal is. All the packages get installed without any errors. Would love to use maizzle on a new project I have coming up.

Thanks.

**Comment by cossssmin** [comment]
Crap, it’s probably the latest update to PostHTML. I’ll publish a patch today so we go back to locking dependency versions, it’s already the second time something like this happens.

**Comment by cossssmin** [comment]
Can confirm, it's `posthtml-expressions`:

```
TypeError: tree.match is not a function
    at scriptDataLocals (C:\Users\Cosmin\Desktop\test-tree\node_modules\posthtml-expressions\lib\locals.js:25:8)
    at C:\Users\Cosmin\Desktop\test-tree\node_modules\posthtml-expressions\lib\index.js:160:24
    at C:\Users\Cosmin\Desktop\test-tree\node_modules\posthtml-extend\lib\index.js:1:2017   
    at Array.reduce (<anonymous>)
    at applyPluginsToTree (C:\Users\Cosmin\Desktop\test-tree\node_modules\posthtml-extend\lib\index.js:1:1989)
    at C:\Users\Cosmin\Desktop\test-tree\node_modules\posthtml-extend\lib\index.js:1:1603   
    at C:\Users\Cosmin\Desktop\test-tree\node_modules\posthtml-extend\node_modules\posthtml\lib\api.js:91:45
    at traverse (C:\Users\Cosmin\Desktop\test-tree\node_modules\posthtml-extend\node_modules\posthtml\lib\api.js:105:26)
    at Array.match (C:\Users\Cosmin\Desktop\test-tree\node_modules\posthtml-extend\node_modules\posthtml\lib\api.js:90:7)
    at handleExtendsNodes (C:\Users\Cosmin\Desktop\test-tree\node_modules\posthtml-extend\lib\index.js:1:1262)
```

@Scrum I suppose this is coming from the latest 1.8.0 release, any ideas? This is how we're applying `posthtml-expressions` in Maizzle:

https://github.com/maizzle/framework/blob/fc768dc2ccdb669f7f88010fecdcb374c9249aea/src/generators/posthtml.js#L33-L43

**Comment by cossssmin** [comment]
@ineptian I published [`3.6.2`](https://github.com/maizzle/framework/releases/tag/v3.6.2), reinstall Maizzle (delete `node_modules` and `package-lock.json`, run `npm install` again) and it'll work.

**Comment by cossssmin** [comment]
Fixed in `posthtml-expressions@1.8.1`, you can update Maizzle to `3.7.0` and it'll work as expected now.

Thanks @Scrum for the speedy fix 🚀
